File: algorithm_data_structures/space_time_complexity/print_remainder_pairs.rb
def print_remainder_pairs(arr)
	for i in 0...arr.size do
		for j in 0...arr.size do
			puts "#{ arr[i] % arr[j]}"
		end
	end
end

arr = [1,2,3,4,5]

print_remainder_pairs(arr)

File: algorithm_data_structures/space_time_complexity/print_remainder_pairs.js
function printRemainderPairs (arr) {
	for (var i = 0; i< arr.length; i++){
		for(var j = 0; j< arr.length ; j++){
			console.log( arr[i] % arr[j]);
		}
	}
}


arr = [1,2,3,4,5]
arr = [42, 2, 12, 5, 7, 9 , 1 , 5]

printRemainderPairs(arr);
